exp-player/addon/styles/components/isp-radio-group.scss

```diff
@@ -24,6 +24,11 @@
   margin-top: -15px;
 }
 
-.label-spacing {
-  max-width: 6%;
+.measure-four {
+  width: 6%;
```

Comment:
To be honest I'm not totally seeing the difference in spacing in the two screenshots.

I'll merge this because we need to go ahead for UCR review, but if there are 11 equally spaced options, then I'm not sure how we arrived at 6% for the final width. (100% / 11 = ~9%)...